credit_integration/enums.py

```diff
@@ -0,0 +1,17 @@
+from django.utils.translation import pgettext_lazy
+from enumfields import Enum
+
+
+class CreditDecisionStatus(Enum):
+    """
+    In Finnish: Luottopäätöksen tila
+    """
+
+    YES = "yes"
+    NO = "no"
+    CONSIDERATION = "consideration"
```

Comment:
Just wondering if this should be "In consideration"/"Harkinnassa" instead of "Consideration"/Harkinta". But if it's been specified like this, it's all good.

Reply:
It should be `Harkinta` so it's correct.